lib/src/dio-interceptor-wrapper.dart

```diff
@@ -168,19 +178,67 @@ class SuperTokensInterceptorWrapper extends Interceptor {
   Future<void> saveTokensFromHeaders(Response response) async {
     String? frontTokenFromResponse =
         response.headers.map[frontTokenHeaderKey]?.first.toString();
+
     if (frontTokenFromResponse != null) {
       await FrontToken.setItem(frontTokenFromResponse);
     }
 
     String? antiCSRFFromResponse =
         response.headers.map[antiCSRFHeaderKey]?.first.toString();
+
     if (antiCSRFFromResponse != null) {
       LocalSessionState localSessionState =
           await SuperTokensUtils.getLocalSessionState();
+
       await AntiCSRF.setToken(
         antiCSRFFromResponse,
         localSessionState.lastAccessTokenUpdate,
       );
     }
+
+    String? accessHeader =
+        response.headers.map[ACCESS_TOKEN_NAME]?.first.toString();
+
+    if (accessHeader != null) {
+      Utils.setToken(TokenType.ACCESS, accessHeader);
+    }
+    String? refreshHeader =
+        response.headers.map[REFRESH_TOKEN_NAME]?.first.toString();
+
+    if (refreshHeader != null) {
+      Utils.setToken(TokenType.REFRESH, refreshHeader);
+    }
+  }
+
+  Future<RequestOptions> _removeAuthHeaderIfMatchesLocalToken(
+      RequestOptions req) async {
+    if (req.headers.containsKey("Authorization") ||
+        req.headers.containsKey("authorization")) {
+      String authValue = req.headers['Authorization'];
```

Comment:
You are checking if it contains either upper case or lower case but you are only fetching upper case inside the if block